wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -39,78 +59,22 @@
+								<p>Test your audience’s knowledge and response rate. Easily gain back-end access to the data from our tools!.</p>
 							</div>
 						</div>
 					</div>
 				</section>
-
-				<section class="callout quiz-creator-research">
-					<div class="container container--xl wrapper--pad">
-
-							<h4 class="subhead">A quiz tool designed to increase civic engagement</h4>
-							<h2 class="headline">Challenge your Audience</h2>
-							Our research shows that quizzes help people learn more than presenting information without a quiz. Even more, they increase time on page and site visitors rate them as enjoyable.
-
-							<a href="/research/online-polls-and-quizzes/">Explore White Paper</a>
 					</div>
 				</section>
-
-				<section class="quiz-creator-steps text-center">
-					<h3 id="startAnchor">Ready, set, go!</h3>
-					<div class="inner-wrapper">
-						<div class="timeline">
-							<div class="bar-container">
-								<div class="progress-bar" id="myBar"></div>
-							</div>
-
-							<div class="events">
-								<div class="event">
-									<div class="event-image" id="stepOneAnchor">
-										<img src="https://mediaengagement.org/wp-content/uploads/2016/08/quiz_creator_step_1@2x.jpg" alt="Create your quiz" />
-									</div><!-- event-image -->
-									<div class="step">
-										<h5 class="step-header">STEP 1</h5>
-										<strong>Create your quiz</strong>
-										<p>Give it a title, add questions and personalize your answer responses.</p>
-									</div><!-- step -->
-								</div><!-- event -->
-
-								<div class="event">
-									<div class="event-image" id="stepTwoAnchor">
-										<img src="https://mediaengagement.org/wp-content/uploads/2017/09/quiz_creator_step_2@2x-300x110.jpg" alt="Customize the color of your quiz" />
-									</div><!-- event-image -->
-									<div class="step">
-										<h5 class="step-header">STEP 2</h5>
-										<strong>Customize</strong>
-										<p>Change the background and font colors and adjust the size to what works best for your site.</p>
-									</div><!-- step -->
-								</div><!-- event -->
-
-								<div class="event">
-									<div class="event-image" id="stepThreeAnchor">
-										<img src="https://mediaengagement.org/wp-content/uploads/2017/09/quiz_creator_step_3@2x-300x110.jpg" alt="Publish your quiz" />
-									</div><!-- event-image -->
-									<div class="step">
-										<h5 class="step-header">STEP 3</h5>
-										<strong>Publish</strong>
-										<p>Share on social media and get an embed code so you can add the quiz to your website.</p>
-									</div><!-- step -->
-								</div><!-- event -->
-							</div><!-- events -->
-
-						</div><!-- timeline -->
-					</div><!-- inner-wrapper -->
-
-					<div class="col-md-6 col-md-offset-3">
-						<a class="btn btn-primary" href="/wp-login.php?action=register">Get Started</a>
 					</div>
 				</section>
+
+
 			</article>
 		</div>
 	</article>
```

Comment:
Unfortunately, we do need to keep `<p>Thanks to <a href="https://www.browserstack.com">BrowserStack</a> for providing us with browser and device testing on our open source tools.</p>` at the end of the file so they'll keep our free account going.